fiftyone/utils/labelstudio.py

```diff
@@ -725,6 +732,30 @@ def _to_detection(label):
     }
     return result, ls_type, label.id
 
+def _to_instance(label, size):
+    ls_type = "brushlabels"
+    if isinstance(label, list):
+        return (
+            [_to_instance(l, size)[0] for l in label],
+            ls_type,
+            [l.id for l in label],
+        )
+
+    if isinstance(label, fol.Detections):
+        return (
+            [_to_instance(l, size)[0] for l in label.detections],
+            ls_type,
+            [l.id for l in label.detections],
+        )
+
+    brush = fou.lazy_import(
+    "label_studio_converter.brush",
```

Comment:
nit: `lazy_import()` can be placed at the very top of the module with other imports, since the module won't actually be imported unless it is used.

I notice that local `lazy_import()` is used elsewhere in this module, perhaps you could clean that up too? 🤗